app/views/runs/_form.html.erb

```diff
@@ -22,18 +22,44 @@
     <div class="field">
       <% if runner_role == 'Coach' %>
       <%= f.label :training_plan %><br>
-      <%= f.text_area :training_plan, rows:"6" %>
+      <%= f.text_area :training_plan, rows:"6", class: 'form-control' %>
       <% end %>
     </div>
     <div class="field">
       <%= f.label :progress %><br>
-      <%= f.text_area :progress, rows:"6" %>
+      <%= f.text_area :progress, rows:"6", class: 'form-control' %>
     </div>
 
+    <div class="field">
+      <% if runner_role == 'Coach' %>
+      <%= f.label :changed_flag, 'Custom' %>
+      <%= f.check_box :changed_flag %>
+      <% end %>
+    </div>
 
     <div class="actions">
-      <%= f.submit %>
+      <%= f.submit 'Update Run', :class => 'btn btn-info' %>
     </div>
 
 
-<% end %>
\ No newline at end of file
+<% end %>
+
+<script>
+  //Javascrip to test if training_plan is different
+  var training_plan;
+
+  $(document).ready(function(){
+      training_plan = $('#run_training_plan').val();
+  });
+
+    $('#run_training_plan').change(function(){
```

Comment:
Ok, I may get my palm smacked for this one but this was the absolute simplest way I could think of to do this without making ajax calls.: https://github.com/o19s/thats-trackable/commit/b625b07f41faab8c823f1fcac61d4a013a2e709e
I just made a hidden p tag with the original planned_run and grabbed it with jQuery for testing if its custom or not. I'd love to hear your thoughts!